src/app/shared/components/fallback-page/fallback-page.component.html

```diff
@@ -12,7 +12,13 @@ <h2 class="text-2xl font-bold mb-4 text-hf-weiches-schwarz">Keine Daten gefunden
 
     <div class="bg-white bg-opacity-50 p-4 rounded-lg border-l-4 border-hf-weiches-schwarz">
       <p class="text-sm text-hf-weiches-schwarz">
-        <strong>Hinweis:</strong> Bitte überprüfen Sie die Eingaben und versuchen Sie es erneut.
+        Wenn Sie sich sicher sind, dass Daten fehlen, melden Sie sich über das Kontaktformular im
+        Footer oder
+        <a
+          href="https://github.com/Hochfrequenz/ahb-tabellen/issues/new"
+          title="erfordert einen Github Account"
+          >öffnen Sie ein Fehlerticket</a
+        >.
```

Comment:
wir versuchen die Leute nicht zu siezen. 


```suggestion
        Bei fehlenden Daten einfach das Kontaktformular im Footer nutzen oder 
        <a
          href="https://github.com/Hochfrequenz/ahb-tabellen/issues/new"
          title="erfordert einen Github Account"
          >ein Fehlerticket öffnen.</a
        >.
```